modules/aidebug/cpp/aidebug.cpp

```diff
@@ -0,0 +1,129 @@
+﻿#include "../src/common/mmo.h"
+#include "../src/map/ai/ai_container.h"
+#include "../src/map/entities/mobentity.h"
+#include "../src/map/entities/npcentity.h"
+#include "../src/map/mob_modifier.h"
+#include "../src/map/utils/mobutils.h"
+#include "../src/map/utils/moduleutils.h"
+#include "../src/map/zone.h"
+#include "../src/map/zone_entities.h"
```

Comment:
All of these shouldn't need to be relative includes, you should be able to use "map/zone.h", I think